pyproject.toml

```
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "qctrl-open-controls"
+name = "open-controls"
 version = "10.0.0"
 description = "Q-CTRL Python Open Controls"
```

Is `description` the same as the project name?
```suggestion
description = "Q-CTRL Open Controls"
```
or 
```suggestion
description = "Open Controls"
```
or

---

Yes, this would be the package name, which hasn't changed. If we remove the `python` from the repo name then it would become "Q-CTRL Open Controls" as suggested.